.gitignore

```diff
@@ -22,6 +22,7 @@ khoj_assistant.egg-info
 /config/khoj*.yml
 .pytest_cache
 khoj.log
+static
```

Comment:
As a one-time operation, Django generates a bunch of static files for admin pages, and also mounts all of the static assets used for our routes (like logo, css files, svgs) into the same directory. As this can be generated with a one-off command, it shouldn't be checked in.